ZenPacks/zenoss/Microsoft/Windows/migrate/RemoveWinRMServices.py

```diff
@@ -21,20 +21,30 @@
 # Zenoss Imports
 from Products.ZenModel.migrate.Migrate import Version
 from Products.ZenModel.ZenPack import ZenPackMigration
+from Products.Zuul.interfaces import ICatalogTool
 
 LOG = logging.getLogger('zen.MicrosoftWindows')
 
 
 class RemoveWinRMServices(ZenPackMigration):
-    version = Version(2, 7, 0)
+    version = Version(2, 7, 1)
 
     def migrate(self, pack):
-        org = pack.dmd.Devices.getOrganizer('/Server/Microsoft')
-        devices = org.getSubDevices()
-        device_count = len(devices)
-        if device_count:
-            LOG.info('Attempting to remove incompatible Windows Services from {} device{}.'
-                     .format(device_count, 's' if device_count > 1 else ''))
-            for device in devices:
-                device.os.removeRelation('winrmservices', suppress_events=True)
+
+        results = ICatalogTool(pack.getDmdRoot("Devices")).search(types=(
+            'ZenPacks.zenoss.Microsoft.Windows.BaseDevice.BaseDevice',
```

Comment:
Have you verified that this does return the expected brains? Sometimes searching the global catalog by type has unexpected results when searching for parent classes. This is because only so many parents up the hierarchy are actually indexed for performance reasons. With multiple inheritance you don't quite know which branches will be followed, and therefore which parent classes will actually be indexed.

So you need to be sure to perform this same search on a system with some Windows devices to make sure they do get returned.